doc/source/management/data_at_rest_encryption.rst

```diff
@@ -79,15 +89,36 @@ To change the default value use the following syntax:
 
 Here, **NEW_ID** is an unsigned 32-bit integer.
 
+
 .. _data-at-rest-encryption.innodb-system-tablespace:
 
+Encryption threads
+========================================================================
+
+Adding explicit encryption in an `ALTER TABLE` or `CREATE TABLE` statement with the `ENCRYPTION`='N' clause excludes the table or tablespace from encryption threads. The INFORMATION_SCHEMA table: INNODB_TABLESPACE_ENCRYPTION displays the information for an excluded table or excluded tablespace. This table is currently used to show information about the tables encrypted by encryption threads. Due to changes in 8.0.16, the `SHOW CREATE TABLE` statement may not verify if the `ENCRYPTION`=’N’ phrase was explicitly specified. 
+
+ MariaDB only allows you to exclude innodb_file_per_table tablespace. In Percona, all tablespaces with `ENCRYPTION`='N' should be excluded from encryption threads. This approach solves the issue of upgrading from 5.7 to 8.0.16. One issue was the explicit encryption would be based on the `CREATE INFO` flags from .frm file or crypt_data must be read before the upgrade. This requirement is not longer with with`INNODB_TABLESPACE_ENCRYPTION`.
```

Comment:
Not sure why we're mentioning MariaDB here (no backwards compatibility unlike MySQL)

Reply:
From discussions, we are illustrating how we implement encryption in a different way than the other dbs.

Reply:
This is great @patrickbirch that you are following our work so closely. However the encryption threads are still in development and those points you are making here are not even yet in review process. They will be part of the server once they are marked as completed.

Reply:
No problem. Re-writing the paragraph to focus on the ENCRYPTION=N information and removing references to threads.

Reply:
The information was removed from this section and checked against the current tablespace encryption section.